cmd/podman/images/load.go

```diff
@@ -71,20 +71,6 @@ func loadFlags(cmd *cobra.Command) {
 }
 
 func load(cmd *cobra.Command, args []string) error {
-	if len(args) > 0 {
-		ref, err := reference.Parse(args[0])
-		if err != nil {
-			return err
-		}
-		if t, ok := ref.(reference.Tagged); ok {
-			loadOpts.Tag = t.Tag()
-		} else {
-			loadOpts.Tag = "latest"
```

Comment:
Shouldn’t the `loadOpts.Tag` field, and similarly all the way down (other than fixed APIs), be removed then, as well? Otherwise we won’t get rid of the ambiguity throughout the codebase.